SecretSanta/test/SecretSanta.Business.Tests/UserManagerTests.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.AspNetCore.Mvc;
+using System.Linq;
+using SecretSanta.Api.Controllers;
+using System.Collections.Generic;
+using SecretSanta.Business;
+using SecretSanta.Data;
+using System;
+
+namespace SecretSanta.Business.Tests
+{
+    [TestClass]
+    public class UserManagerTests
+    {
+        [TestMethod]
+        public void Create_WithData_ReturnsUser(int id)
+        {
+            //Arrange
+            UserRepository repo = new();
+
+            //Act
+            User user = new();
+            user.Id = id;
+
+            //Assert
+            Assert.AreEqual(repo.Create(id), user);
+        }
```

Comment:
Nit: not a big deal but some of the code is under the wrong comments. Something like this would be more accurate with what is happening.
```suggestion
        public void Create_WithData_ReturnsUser(int id)
        {
            //Arrange
            UserRepository repo = new();
            User user = new();
            user.Id = id;
            
            //Act /Assert
            Assert.AreEqual(repo.Create(id), user);
        }
```